changelogs/current.yaml

```diff
@@ -2,6 +2,13 @@ date: Pending
 
 behavior_changes:
 # *Changes that are expected to cause an incompatibility if applicable; deployment changes are likely required*
+- area: eds
+  change: |
+    Introduced caching of EDS assignments when used with ADS. Without this change, Envoy required EDS assignments to always
+    be sent after a cluster was updated. Following a cluster upadate, Envoy will wait for an EDS assignment until
+    :ref:`initial_fetch_timeout <envoy_v3_api_field_config.core.v3.ConfigSource.initial_fetch_timeout` times out, and then apply
```

Comment:
```suggestion
    :ref:`initial_fetch_timeout <envoy_v3_api_field_config.core.v3.ConfigSource.initial_fetch_timeout>` times out, and then apply
```

typo

Reply:
Oops :face_palm:, thanks for pointing it out!